packages/calcite-components/src/components/handle/handle.scss

```diff
@@ -1,3 +1,17 @@
+/**
+* CSS Custom Properties  
+*
+* These properties can be overridden using the component's tag as selector.
+*
+* @prop --calcite-handle-background-color: Specifies the component's background color.
+* @prop --calcite-handle-background-color-hover: Specifies the component's background color on hover.
+* @prop --calcite-handle-background-color-selected: Specifies the component's background color when selected.
+* @prop --calcite-handle-icon-color: Specifies the component's icon color.
+* @prop --calcite-handle-icon-color-hover: Specifies the component's icon color on hover.
+* @prop --calcite-handle-icon-color-pressed: Specifies the component's icon color when pressed.
```

Comment:
Do we need `--calcite-handle-icon-color-pressed`? According to the [Component Tokens](https://github.com/Esri/calcite-design-system/wiki/Component-Tokens/094b0ad46e6701adf9d0c4436357a6a4ba27d0eb#naming-pattern) doc, it doesn't qualify under the following guideline:

> A state shared by :focus and :active should use -pressed

Worth noting that the background doesn't have an associated pressed prop either.

Reply:
yes this should be left off

Reply:
Looks like it doesn't.